build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 VERSION="1.1.3"
-MILESTONE=master
+MILESTONE=
 RPM_RELEASE="1.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
 
```

Comment:
Shouldn't the RPM release be "1" as this is official release?

Reply:
sure, done